quotesimulator/quote_simulator.go

```diff
@@ -67,16 +62,10 @@ func (q *quoteSimulator) SimulateQuote(ctx context.Context, quote domain.Quote,
 	// Get the account for the simulator address
 	baseAccount, err := q.accountQueryClient.GetAccount(ctx, simulatorAddress)
 	if err != nil {
-		return 0, sdk.Coin{}, err
-	}
-
-	// Price the message
-	gasAdjusted, feeCoin, err := q.msgSimulator.PriceMsgs(ctx, q.txFeesClient, q.encodingConfig.TxConfig, baseAccount, q.chainID, swapMsg)
-	if err != nil {
-		return 0, sdk.Coin{}, err
+		return domain.QuotePriceInfo{Err: err.Error()}
```

Comment:
Will this return an empty Error struct/string, causing the caller consider it is a valid quote price data to be used? Suggest to have some meaning error message in it.

Reply:
I don't understand the concern, This returns a struct with an error embedded that the caller can check